clients/cli/build.gradle.kts

```diff
@@ -67,3 +67,14 @@ tasks.jar {
   from(dependencies)
   duplicatesStrategy = DuplicatesStrategy.EXCLUDE
 }
+
+tasks.test {
+  val skipITs = project.hasProperty("skipITs")
+  if (skipITs) {
+    // Exclude integration tests
+    exclude("**/integration/test/**")
+  } else {
+    dependsOn(tasks.jar)
```

Comment:
unnecessary?

Reply:
hi @mchades 
We need to add this depends,
because clients-cli need to compile a `Main-Class` jar.
https://github.com/apache/gravitino/blob/main/clients/cli/build.gradle.kts#L59